### DISCUSSION TITLE
French Language Adaptation

### DISCUSSION THREAD

**Discussion by Compr0mzd** [outline/outline#5385] [Ideas]
Hello,,

When we write using proper French grammar it's is really an inconvenient to have the emojis proposition each time you add the ":" 
For example it is a proper way of writting in french to write ":" like this: 
"Exemple : numéro 1" 
 So each time you add the ":" it's proposing the emojis since there is a space between the last word. Is there any possibility to check if the language is french and change this behavior of the emojis panel  ? 

I hope it was clear enough 
Thanks in Advance

**Comment by tommoor** [comment]
I think the emoji suggestions should close if you type a space character after `:` – would this work or should we be looking at preventing the menu opening until another character is typed? How does this work elsewhere?

**Reply by Compr0mzd** [reply]
Yes closing the emoji button after you press spacebar is a behavior that is common. On Github it is the same behavior. 
It would be a great way to prevent this slight discomfort when typing.

**Reply by Compr0mzd** [reply]
> I think the emoji suggestions should close if you type a space character after `:` – would this work or should we be looking at preventing the menu opening until another character is typed? How does this work elsewhere?

Hey I saw that you updated the code ASAP about this issue it's really neat. Do you know when it will be pushed to get the modification ? 
Thanks in advance
